### ISSUE TITLE
Stopping ExpressServer in combination with MemoryStore

### ISSUE THREAD

**Issue by haeferer** [roccomuso/memorystore#31]
Hi, 

thanx for your great work. 
I would like to suggest to add a small Info to the documentation.

If you want to close the server completly, remember to call  stopInterval on the Store instance. 
Otherwise the node process will not end (cause of the still active interval)

The current code-sample is small, but a litte bit complicated, cause the store instance is not accessible. 

```
var session = require('express-session')
var MemoryStore = require('memorystore')(session)

app.use(session({
    cookie: { maxAge: 86400000 },
    store: new MemoryStore({
      checkPeriod: 86400000 // prune expired entries every 24h
    }),
    resave: false,
    secret: 'keyboard cat'
}))
```
a better version would be
```
var session = require('express-session')
var MemoryStore = require('memorystore')(session)
let store = new MemoryStore({
      checkPeriod: 86400000 // prune expired entries every 24h
});
app.use(session({
    cookie: { maxAge: 86400000 },
    store: store,
    resave: false,
    secret: 'keyboard cat'
}))

// call store.stopInterval() if you want to shut down the memorystore
```

**Comment by roccomuso** [comment]
We can add `.unref()` to the interval to avoid blocking the process.

I'm gonna push both the improvements 👍🏼

**Comment by haeferer** [comment]
Would be nice.

thx

**Comment by roccomuso** [comment]
fixed in `v1.6.7`
